README.md

```diff
@@ -1,492 +1,271 @@
     {
-      //Add code to go an inject or insert the tester data
+        this.fixture = fixture;
+        this.output = output;
     }
 
-    public async Task Invoke(IDictionary<string, object> environment)
+    [Fact]
+    public void EnsureSomethingApiHonoursPactWithConsumer()
     {
-        IOwinContext context = new OwinContext(environment);
-
-        if (context.Request.Path.Value == "/provider-states")
+        //Arrange
+        var config = new PactVerifierConfig
         {
-            context.Response.StatusCode = (int)HttpStatusCode.OK;
-
-            if (context.Request.Method == HttpMethod.Post.ToString() &&
-                context.Request.Body != null)
+            Outputters = new List<IOutput>
             {
-                string jsonRequestBody;
-                using (var reader = new StreamReader(context.Request.Body, Encoding.UTF8))
-                {
-                    jsonRequestBody = reader.ReadToEnd();
-                }
-
-                var providerState = JsonConvert.DeserializeObject<ProviderState>(jsonRequestBody);
-
-                //A null or empty provider state key must be handled
-                if (providerState != null &&
-                    !IsNullOrEmpty(providerState.State) &&
-                    providerState.Consumer == ConsumerName)
-                {
-                    _providerStates[providerState.State].Invoke();
-                }
-
-                await context.Response.WriteAsync(Empty);
-            }
-        }
-        else
-        {
-            await m_next.Invoke(environment);
-        }
-    }
-}
-```
-
-```c#
-public class ProviderState
-{
-    public string Consumer { get; set; }
-    public string State { get; set; }
-}
-```
+                // NOTE: We default to using a ConsoleOutput,
+                // however xUnit 2 does not capture the console
+                // output, so a custom outputter is required.
+                new XUnitOutput(output),
+            },
+        };
 
-```c#
-public class XUnitOutput : IOutput
-{
-    private readonly ITestOutputHelper _output;
+        string pactPath = Path.Combine("..",
+                                        "..",
+                                        "path",
+                                        "to",
+                                        "pacts",
+                                        "Something API Consumer-Something API.json");
 
-    public XUnitOutput(ITestOutputHelper output)
-    {
-        _output = output;
-    }
+        // Thread.Sleep(TimeSpan.FromMinutes(1));
 
-    public void WriteLine(string line)
-    {
-        _output.WriteLine(line);
+        // Act / Assert
+        IPactVerifier pactVerifier = new PactVerifier(config);
+        pactVerifier
+            .ServiceProvider("Something API", fixture.ServerUri)
+            .HonoursPactWith("Something API Consumer")
+            .FromPactFile(new FileInfo(pactPath))

```

Comment:
The original docs showed how you've got a few different options for this, such as using Pact Broker to retrieve your pacts instead of from a local file. I think it would be useful to keep that because we don't document those sorts of things anywhere else.

Reply:
I'd recommend making this example point to a Pact Broker with a good default configuration, because people will almost certainly copy paste.

I say that knowing full well the Golang example doesn't do this, and I'm going to fix that.

Doesn't need to be in this PR though.